tuxemon/formula.py

```diff
@@ -215,6 +216,99 @@ def escape(level_user: int, level_target: int, attempts: int) -> bool:
         return False
 
 
+def today_ordinal() -> int:
+    """
+    It gives today's proleptic Gregorian ordinal.
+    """
+    today = dt.date.today().toordinal()
+    return today
+
+
+def set_capture() -> int:
```

Comment:
Is this still used?

Reply:
Yes, it's used for all the existing monster, to give them a random capture date (one of the last 1000 days); eventually we can remove it and assign "today" date.

Reply:
Look, let's assign "today" date, I'll remove formula.doc too, at the moment it's not used, the day I'll find something useful I'll add again.